sw/device/sca/aes_serial.c

```diff
@@ -39,6 +38,34 @@ enum {
   kIbexAesSleepCycles = 400,
 };
 
+/**
+ * Max number of encryption that can be captured with the scope
+ * 81 is selected for AES with CW Husky
+ * Note: Maybe it would be better if we use dynamic memory allocation but I
+ * am not sure whether we are supporting it or not.
+ */
+#define MAX_NUM_OF_OPS_IN_A_BATCH 81
```

Comment:
Can you move this to the enum above?

Also, this doesn't seem to be checked anywhere. I'd expect something like the following in `aes_serial_fvsr_key_batch_encrypt()`:

```
SS_CHECK(num_encryptions <= kNumBatchOpsMax);
```